lxdock/container.py

```diff
@@ -47,8 +69,12 @@ class Container:
     def __init__(self, project_name, homedir, client, **options):
         self.project_name = project_name
         self.homedir = homedir
-        self.client = client
         self.options = options
+        self.lxd_host = os.path.basename(get_container_host(self.lxd_name)).split(':')[0]
+        try:
+            self.client = get_client([get_container_host(self.lxd_name)])
+        except:
```

Comment:
Catching all exceptions should only be used as a last resort.

Reply:
good catch, I just had that there for debug purpose